css/main.css

```diff
@@ -131,4 +131,20 @@ p {
 .addedInput{
     width: 110px;
     text-align: center;
-}
\ No newline at end of file
+}
+
+table {
```

Comment:
don't use styles for elements, use classes
what if one more table will be added